src/btree/bt_vrfy.c

```diff
@@ -336,6 +337,24 @@ __verify_addr_ts(WT_SESSION_IMPL *session, WT_REF *ref, WT_CELL_UNPACK *unpack,
           "transaction (%" PRIu64 ")",
           __wt_page_addr_string(session, ref, vs->tmp1), unpack->oldest_start_txn,
           unpack->newest_stop_txn);
+    if (parent_unpack->oldest_start_ts > unpack->oldest_start_ts)
+        WT_RET_MSG(session, WT_ERROR,
+          "child page reference at %s has oldest start timestamp %s"
+          "older than internal parent page reference at %s"
+          "oldest start timestamp %s",
+          __wt_page_addr_string(session, ref, vs->tmp1),
```

Comment:
What should the message be like? Is this sufficient enough? Let me know :)

Reply:
I'm happy with this msg.